lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -100,8 +106,15 @@ public function dropAndCreateForeignKey(ForeignKeyConstraint $foreignKey, $table
      */
     public function dropForeignKey($foreignKey, $table) : void
     {
-        $tableDiff                       = $this->getTableDiffForAlterForeignKey($table);
-        $tableDiff->removedForeignKeys[] = $foreignKey;
+        $table = $this->ensureTable($table);
```

Comment:
These changes were made to fix this type https://github.com/doctrine/dbal/pull/3565/commits/125265bd436523fee013e38c7e57d29b3c275995#diff-6b78caddb762256c1425fe6c316e07dcL93